web/web/api/v1/sns_read.py

```diff
@@ -0,0 +1,30 @@
+from flask import Flask, request, Blueprint
+import json
+
+sns_read_api_bp =  Blueprint('sns_read_api_bp', __name__)
+
+
+@sns_read_api_bp.route('/sns_read', methods = ['GET', 'POST'])
+def sns():
```

Comment:
Should this be called `confirm_subscription` instead?

Reply:
I realized this receives the confirmation and the message from the devices. Maybe we can call this `subscriber_information` or `listener`, or something of the sorts

Reply:
done